Riot/Modules/Home/AllChats/AllChatsViewController.swift

```diff
@@ -181,7 +186,8 @@ class AllChatsViewController: HomeViewController {
         AppDelegate.theDelegate().checkAppVersion()
 
         if BuildSettings.newAppLayoutEnabled && !RiotSettings.shared.allChatsOnboardingHasBeenDisplayed {
-            self.showAllChatsOnboardingScreen()
+            // Tchap: Not the same onboarding flow.
```

Comment:
Please update the comment with
 // Tchap: Disable new layout tutorials. Wait for an updated tutorials before show it

Reply:
Fixed in last commit